examples/undocumented/python/distance_sparseeuclidean.py

```diff
@@ -1,26 +0,0 @@
-#!/usr/bin/env python
-import shogun as sg
-traindat = '../data/fm_train_sparsereal.dat'
-testdat = '../data/fm_test_sparsereal.dat'
-
-parameter_list = [[traindat,testdat],[traindat,testdat]]
-
-def distance_sparseeuclidean (train_fname=traindat,test_fname=testdat):
-	from shogun import SparseRealFeatures
-	import shogun as sg
-
-	feats_train=SparseRealFeatures(sg.libsvm_file(train_fname))
-	feats_test=SparseRealFeatures(sg.libsvm_file(test_fname))
```

Comment:
Why don’t you use this parser and files?

Reply:
Won't I have to add functions to the `factory.h` file nevertheless? so that it can be translated?